test.js

```diff
@@ -0,0 +1,79 @@
+const buttons=document.querySelectorAll('button');
+const rock=document.querySelectorAll('rock');
+const computerChoice=document.querySelector('.computerChoice');
+const userChoice=document.querySelector('.yourChoice');
+const winner=document.querySelector('.result');
+const score=document.querySelector('.score');
+const reset=document.getElementById("reset");

```

Comment:
✨P1: html에서는 reset이 class이므로, querySelector를 사용해야합니다.
```suggestion
const resetBtn = document.querySelector(".reset");
```